### DISCUSSION TITLE
Can I use more than 1 bot in raspberrypi

### DISCUSSION THREAD

**Discussion by HaiPingCao** [jagrosh/MusicBot#1009] [Questions]
Can I use more than 1 bot in raspberrypi

**Comment by ghost** [comment]
My suggestion would be to use docker containers. Just create 2 different bots in the discord dev portal. Then run 2 docker containers. Edit the configs accordingly so that each container has its own bot token. I do the same and it works like a charm.

**Comment by Wyvest** [comment]
you can. for example with this you could literally just open the jar file and a copy of the jar file
